Documentation/sections/Documentation.md

```diff
@@ -17,10 +17,11 @@ By integrating this SDK into your application you can allow your users to easily
 
 Further documentation with installation, integration or customization guides can be found in our [website](http://developer.gini.net/gini-pay-bank-sdk-ios/docs/).
 
-## Example
+## Example apps
 
-We are providing example app for integrating payment functionality for Swift. This app demonstrates how to integrate the Gini Pay Bank SDK. To run the example project, clone the repo and run `pod install` from the Example directory first.
-To inject your API credentials into the Example app, just add to the Example/Bank directory the `Credentials.plist` file with the following format:
+We are providing Bank and Business example apps for integrating payment functionality. Bank app demonstrates how to integrate the Gini Pay Bank SDK. Business app initiates the payment flow.
+To run the apps, clone the repo and run `pod install` from the Example directory first.
+To inject your API credentials into Business and Bank, just add to the Example/Bank and Example/Business directory the `Credentials.plist` file with the following format:
```

Comment:
Less German :) `add the `Credentials.plist` file to the Example/Bank and Example/Business directory. The file should follow the format:`

Reply:
Could you add these Credentials.plist files with placeholder content? Then clients would need to only fill it out, like [here](https://github.com/gini/gini-pay-business-sdk-ios/blob/master/Example/Example%20Swift/Credentials.plist). Then we also don't need the format information here.
```
To inject your API credentials into the Business and Bank example apps you need to add your credentials to `Example/Business/Credentials.plist` and `Example/Bank/Credentials.pllist`, respectively.
```

Reply:
files already there, I'll update the text